src/main/java/org/wso2/carbon/mongodb/util/MongoDatabaseUtil.java

```diff
@@ -78,6 +75,11 @@ public static synchronized DB getRealmDataSource(RealmConfiguration realmConfigu
      * @throws UserStoreException if any error occurred
      */
     public static DB createRealmDataSource(RealmConfiguration realmConfiguration) throws UserStoreException {
+
+        if (log.isDebugEnabled()) {
+            log.debug("createRealmDataSource(RealmConfiguration realmConfiguration)");
```

Comment:
can we change this to "Creating mongo datasource with realm configuration "+ realmConfiguration+getDescription